coderd/database/queries/user_links.sql

```diff
@@ -78,3 +77,36 @@ WHERE
 		ELSE true
 	END
 ;
+
+-- name: OIDCClaimFieldValues :many
```

Comment:
Where is this being / going to be called?

Reply:
In the `Mappings` section, this generates the left hand side options. 

So if doing Group sync, this shows the user which IDP groups exist.